keyboards/framework/laptop16/framework.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022-2023 Framework Computer
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+enum framework_keycodes {
+    // Function lock button
+    FN_LOCK = SAFE_RANGE,
```

Comment:
`SAFE_RANGE` is meant for users, this starts at the dedicated keyboard block:
```suggestion
    FN_LOCK = QK_KB,
```

Reply:
Done, thanks!